.travis.yml

```diff
@@ -99,8 +99,8 @@ install:
   - "mkdir -p $TRAVIS_BUILD_DIR/build/release"
   # Enable test coverage for travis-ci build
   - "cmake $TRAVIS_BUILD_DIR -DNTA_COV_ENABLED=ON -DCMAKE_INSTALL_PREFIX=${TRAVIS_BUILD_DIR}/build/release -DPY_EXTENSIONS_DIR=$TRAVIS_BUILD_DIR/bindings/py/nupic/bindings"
-  - "VERBOSE=1 make"
-  - "make install"
+  - "VERBOSE=1 make|grep -v -F '\\-\\- Installing:'"
+  - "make install 2>&1|grep -v -F 'Installing:'"
```

Comment:
Good catch.

@vitaly-krugl Do you know how to "configure bash such that a failure in any link in the pipe causes the result of the pipe to be deemed as failure"?

According to [bash doc about pipeline](http://www.gnu.org/software/bash/manual/bash.html#Pipelines), it seems `set -o pipefail` would do the trick?